content/apidocs-mxsdk/apidocs/app-repository-api.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "App Repository API"
+category: "API Documentation"
+menu_order: 65
+---
+
+## 1 Introduction
+
+The App Repository API enables retrieving information (branches, commits) of application models stored in our team server.
```

Comment:
'team server' --> 'Team Server'

Also, let's make it a link to the Team Server documentation page (in `developerportal/collaborate/team-server`)